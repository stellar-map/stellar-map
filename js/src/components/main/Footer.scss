.root {
  display: flex;
  justify-content: center;
  padding: 30px 0;
  background-color: lightgray;
}

.copyright {
  font-size: 10px;
  color: gray;
}
