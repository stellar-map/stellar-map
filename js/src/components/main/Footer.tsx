import * as React from 'react';

import RegularContainer from '../../ui/RegularContainer';

import typography from '../../typography.scss';
import styles from './Footer.scss';

const Footer: React.SFC = () => (
  <section className={styles.root}>
    <RegularContainer>
      <div className={`${styles.copyright} ${typography.textXsmall}`}>
        &copy; 2018 StellarMap
      </div>
    </RegularContainer>
  </section>
);

export default Footer;
