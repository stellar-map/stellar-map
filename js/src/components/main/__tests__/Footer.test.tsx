import * as React from 'react';

import { shallow } from 'enzyme';

import Footer from '../Footer';

describe('Footer', () => {
  it('renders correctly', () => {
    expect(shallow(
      <Footer />,
    )).toMatchSnapshot();
  });
});
