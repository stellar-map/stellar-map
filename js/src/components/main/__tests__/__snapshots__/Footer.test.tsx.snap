// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="root"
>
  <RegularContainer>
    <div
      className="copyright text-xsmall"
    >
      Â© 2018 StellarMap
    </div>
  </RegularContainer>
</div>
`;
